### PR TITLE
Get user input

### PR DESCRIPTION
**Pull request by mkrump**
https://chicagotechgrowhouse.atlassian.net/browse/MKA-5

### CODE REVIEW

File: src/tictactoe/board.clj [hunk 1]
```diff
@@ -0,0 +1,27 @@
+(ns tictactoe.board
+  (:require [clojure.set :as set]
+            [clojure.string :as s]))
```
Comment (andarcabrera) [inline src/tictactoe/board.clj]: `s` is not very descriptive

File: src/tictactoe/ui.clj [hunk 2]
```diff
@@ -0,0 +1,14 @@
+(ns tictactoe.ui
+  (:require [tictactoe.board :as rb])
```
Comment (andarcabrera) [inline src/tictactoe/ui.clj]: Same for `rb`. I like aliases to shorten the full name, but I think they should still be descriptive

Comment (mkrump) [reply src/tictactoe/ui.clj]: Updated names to be more descriptive in 	18ef814

File: src/tictactoe/board.clj (hunk 1 above)
Comment (mkrump) [reply src/tictactoe/board.clj]: Updated names to be more descriptive in 	18ef814

**Pull request merged**